tests/test_dict.py

```diff
@@ -18,6 +18,10 @@
 import os
 import shutil
 import unittest
+from parlai.core.build_data import download
+
+DEFAULT_BYTELEVEL_BPE_VOCAB = 'https://dl.fbaipublicfiles.com/parlai/test_bytelevel_bpe_v1/test-byte-level-bpe-v1-vocab.json'
```

Comment:
These should be placed in a `parlai/_models/unittests/test_bytelevel_bpe_v1.tar.gz` and moved into the build statement in `parlai/zoo/unittests/build.py`. Then you can refer to them as `zoo:unittests/bytelevel_bpe_v1/convai2-vocab.json` and zoo:unittests/bytelevel_bpe_v1/convai2-merges.txt`.

Reply:
It should be unittest instead of unittests I think.